buildpipeline/dotnet-standard-win.json

```diff
@@ -18,6 +18,25 @@
         "feedSource": "https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json"
       }
     },
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Run script $(VS140COMNTOOLS)\\VsDevCmd.bat",
```

Comment:
Why is this not needed in master?

Reply:
It is - https://github.com/dotnet/standard/pull/730

Reply:
I was mistaken about a build being queued automatically in master, the last build there ran this morning.

Reply:
Lets not do it this way. Lets do it similar to build.cmd and let run.cmd handle the initialization of VS.

I suggest adding a target alias in config.json and just call it.  Something similar to https://github.com/dotnet/standard/blob/master/config.json#L123.

Reply:
Would it not also work to just call `Tools/msbuild.cmd` instead of trying to invoke msbuild directly? As in https://github.com/dotnet/standard/blob/master/config.json#L198

Reply:
That might work and would be acceptable as well. I mainly want to try and avoid setting up VS as I want to stop depending on it over-time.